pkg/kubelet/dockershim/docker_container.go

```diff
@@ -438,3 +447,20 @@ func (ds *dockerService) UpdateContainerResources(_ context.Context, r *runtimea
 	}
 	return &runtimeapi.UpdateContainerResourcesResponse{}, nil
 }
+
+func (ds *dockerService) performPlatformSpecificContainerCleanupForContainer(containerID string) {
+	if cleanupInfo, present := ds.containerCleanupInfos[containerID]; present {
+		ds.performPlatformSpecificContainerCleanupAndLogErrors(containerID, cleanupInfo)
+		delete(ds.containerCleanupInfos, containerID)
+	}
+}
+
+func (ds *dockerService) performPlatformSpecificContainerCleanupAndLogErrors(containerNameOrID string, cleanupInfo *containerCleanupInfo) {
+	if cleanupInfo == nil {
+		return
+	}
+
+	for _, err := range ds.performPlatformSpecificContainerCleanup(cleanupInfo) {
+		klog.Warningf("error when cleaning up after container %q's: %v", containerNameOrID, err)
```

Comment:
The `'s` can be removed in the message after `container %q's`

Reply:
Oops. Typo.